draft-ietf-rtcweb-jsep.xml

```diff
@@ -3163,6 +3164,935 @@ a=end-of-candidates
+            raddr 192.168.1.2 rport 51556
+a=candidate:3671762466 1 udp 41819903 22.33.44.55 61405 typ relay
+            raddr 11.22.33.44 rport 52546
+a=end-of-candidates
+
+m=application 52546 UDP/DTLS/SCTP webrtc-datachannel
+c=IN IP4 11.22.33.44
+a=mid:d1
+a=fmtp:webrtc-datachannel max-message-size=65536
+a=sctp-port 5000
+a=ice-ufrag:ATEn1v9DoTMB9J4r
+a=ice-pwd:AtSK0WpNtpUjkY4+86js7ZQl
+a=ice-options:trickle
+a=fingerprint:sha-256 19:E2:1C:3B:4B:9F:81:E6:B8:5C:F4:A5:A8:D8:73:04
+                     :BB:05:2F:70:9F:04:A9:0E:05:E9:26:33:E8:70:88:A2
+a=setup:passive
+a=candidate:109270923 1 udp 2122194687 192.168.1.2 51556 typ host
+a=candidate:4036177503 1 udp 1685987071 11.22.33.44 52546 typ srflx
+            raddr 192.168.1.2 rport 51556
+a=candidate:3671762466 1 udp 41819903 22.33.44.55 61405 typ relay
+            raddr 11.22.33.44 rport 52546
+a=end-of-candidates
+
+m=video 52546 UDP/TLS/RTP/SAVPF 100 101
+c=IN IP4 11.22.33.44
+a=rtcp:52546 IN IP4 11.22.33.44
+a=mid:v1
+a=recvonly
+a=rtpmap:100 VP8/90000
+a=rtpmap:101 rtx/90000
+a=fmtp:101 apt=100
+a=ice-ufrag:ATEn1v9DoTMB9J4r
+a=ice-pwd:AtSK0WpNtpUjkY4+86js7ZQl
+a=ice-options:trickle
+a=fingerprint:sha-256
+              19:E2:1C:3B:4B:9F:81:E6:B8:5C:F4:A5:A8:D8:73:04
+             :BB:05:2F:70:9F:04:A9:0E:05:E9:26:33:E8:70:88:A2
+a=setup:passive
+a=rtcp-mux
+a=rtcp-rsize
+a=extmap:2 urn:ietf:params:rtp-hdrext:sdes:mid
+a=rtcp-fb:100 ccm fir
+a=rtcp-fb:100 nack
+a=rtcp-fb:100 nack pli
+a=candidate:109270923 1 udp 2122194687 192.168.1.2 51556 typ host
+a=candidate:4036177503 1 udp 1685987071 11.22.33.44 52546 typ srflx
+          raddr 192.168.1.2 rport 51556
+a=candidate:3671762466 1 udp 41819903 22.33.44.55 61405 typ relay
+          raddr 11.22.33.44 rport 52546
+a=end-of-candidates
+
+m=video 52546 UDP/TLS/RTP/SAVPF 100 101
+c=IN IP4 11.22.33.44
+a=rtcp:52546 IN IP4 11.22.33.44
+a=mid:v2
+a=recvonly
+a=rtpmap:100 VP8/90000
+a=rtpmap:101 rtx/90000
+a=fmtp:101 apt=100
+a=ice-ufrag:ATEn1v9DoTMB9J4r
+a=ice-pwd:AtSK0WpNtpUjkY4+86js7ZQl
+a=ice-options:trickle
+a=fingerprint:sha-256
+              19:E2:1C:3B:4B:9F:81:E6:B8:5C:F4:A5:A8:D8:73:04
+             :BB:05:2F:70:9F:04:A9:0E:05:E9:26:33:E8:70:88:A2
+a=setup:passive
+a=rtcp-mux
+a=rtcp-rsize
+a=extmap:2 urn:ietf:params:rtp-hdrext:sdes:mid
+a=rtcp-fb:100 ccm fir
+a=rtcp-fb:100 nack
+a=rtcp-fb:100 nack pli
+a=candidate:109270923 1 udp 2122194687 192.168.1.2 51556 typ host
+a=candidate:4036177503 1 udp 1685987071 11.22.33.44 52546 typ srflx
+            raddr 192.168.1.2 rport 51556
+a=candidate:3671762466 1 udp 41819903 22.33.44.55 61405 typ relay
+            raddr 11.22.33.44 rport 52546
+a=end-of-candidates
+         ]]></artwork>
+       </figure>
+     </t>
+    </section>
+    -->
+
+    <section title="Inactive Media Answer Example" anchor="sec.inactive-examples" >
+
+      <t> In this example, browser A sends an offer to browser
+      B with an offer for data, audio, and two video tracks. Browser B immediately
```

Comment:
Having two video tracks is probably unnecessary complexity. I even wonder if we should include the data channel here.